pkg/networkservice/chains/nsmgr/heal_test.go

```diff
@@ -278,7 +278,7 @@ func TestNSMGRHealEndpoint_DatapathHealthy_CtrlPlaneBroken(t *testing.T) {
 	domain.Nodes[0].NewEndpoint(ctx, nseReg2, sandbox.GenerateTestToken, counter)
 
 	// Should not connect to new NSE
-	require.Eventually(t, func() bool { return counter.UniqueRequests() == 1 }, timeout, tick)
+	require.Never(t, func() bool { return counter.UniqueRequests() > 1 }, time.Second*2, tick)
```

Comment:
Why do we need to change this?

Reply:
This is an invalid condition in the test. It only worked on ci because of the timeout settings.

This test checks that if the dataplan is alive, we will not select another endpoint. Therefore, the number of unique requests to endpoints should never exceed 1.

In the previous implementation, the exit from `Eventually` happened almost instantly, just without waiting for the second request.